west.yml

```diff
@@ -102,7 +102,9 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 0ae93a2f7da1b65e8b0c1f61849b78910183fc32
+
+      revision: pull/340/head
+
```

Comment:
please remove the additional empty lines when updating to a SHA.

Reply:
> please remove the additional empty lines when updating to a SHA.

Yes. This is done by my mergetool for some reason. Will fix